module-system/module-implementations/sov-ibc/src/context/clients.rs

```diff
@@ -314,14 +314,28 @@ where
         client_id: &ClientId,
         height: &ibc::Height,
     ) -> Result<Option<Self::AnyConsensusState>, ContextError> {
-        todo!()
+        let client_cons_state_path = ClientConsensusStatePath::new(client_id, &height.increment());
+
+        let next_cons_state = self
+            .ibc
+            .consensus_state_map
+            .get(&client_cons_state_path, *self.working_set.borrow_mut());
+
+        Ok(next_cons_state)
     }
 
     fn prev_consensus_state(
         &self,
         client_id: &ClientId,
         height: &ibc::Height,
     ) -> Result<Option<Self::AnyConsensusState>, ContextError> {
-        todo!()
+        let client_cons_state_path = ClientConsensusStatePath::new(client_id, &height.decrement()?);
```

Comment:
Similar comment to [this one](https://github.com/informalsystems/sovereign-sdk/pull/4/files#r1327422729)